### DISCUSSION TITLE
Deployment on AWS guidelines

### DISCUSSION THREAD

**Discussion by detinho** [apache/doris#34389] [RFC]
I tried to find some guidelines for deploying doris on AWS but could not find any detailed information, espcially regarding on hardware requeriments.

On the [K8S operator page](https://github.com/selectdb/doris-operator) there are some requeriments about memory but that is it.

* What is some typical hardware recommendation for running doris on AWS?
* Should we use EBS or instance stores/disks? Any advantage or disvantage of one over another?
* Can we deploy on a multi AZ configuration?
* Any specific instance type recommendation?
* For long running clusters, is it feasible to use spot instances? Or a mix of spot and on demand instances?

Thanks a lot!

**Comment by yagagagaga** [comment]
- What is some typical hardware recommendation for running doris on AWS?
   > This depends on the size of your data. It's recommend using machines with specifications above 16c64g.
- Should we use EBS or instance stores/disks? Any advantage or disvantage of one over another?
  > This depends on the specification parameters of both. The better the disk performance, the better the Doris performance.
- Can we deploy on a multi AZ configuration?
  > Not recommend.
- Any specific instance type recommendation?
  > 
- For long running clusters, is it feasible to use spot instances? Or a mix of spot and on demand instances?
  > Don't use spot instances.

**Reply by detinho** [reply]
Thanks for the reply!

Is there any reason for not using multi AZ? Are there any plans the roadmap to address this?

**Reply by yagagagaga** [reply]
Multi AZ will cause network latency and may incur some traffic costs.
